yb-voyager/cmd/export.go

```diff
@@ -53,6 +53,8 @@ func init() {
 func registerCommonExportFlags(cmd *cobra.Command) {
 	BoolVar(cmd.Flags(), &startClean, "start-clean", false,
 		"cleans up the project directory for schema or data files depending on the export command (default false)")
+
+	BoolVar(cmd.Flags(), &source.RunGuardrailsChecks, "run-guardrails-checks", true, "run guardrails checks before exporting data")
```

Comment:
@ShivanshGahlot Please set it as false as of now.

Reply:
Yep will change it before merging. Kept it true for testing.